master/internal/resourcemanagers/scheduler_test.go

```diff
@@ -354,15 +355,16 @@ func setupSchedulerStates(
 
 		groups[ref] = &group{handler: ref}
 
-		var jobID *model.JobID
+		var jobID model.JobID
 		if mockTask.jobID != "" {
 			jid := model.JobID(mockTask.jobID)
-			jobID = &jid
+			jobID = jid
 		}
 
 		req := &sproto.AllocateRequest{
 			AllocationID:      mockTask.id,
 			JobID:             jobID,
+			IsUserVisible:     true,
```

Comment:
I have a small refactor in another pr to reuse how we go from mockTask to AllocateRequest